README.md

```diff
@@ -111,140 +107,45 @@ It is fairly simple for a third party to constatly poll a given alias's endpoint
 
 <hr>
 
-## ENDPOINTS
-
-All endpoints (Tier 1 and Tier 2) adhere to RESTful standards and communicate using JSON. 
-
-All endpoints are served at the URL in the following format:
+## Connecting to the CAP server
 
-https://opencap.{domain}.{extension}:443/{endpoint}
+When a wallet gets an alias like john@doe.com it first has to determine which server to query.  
+It will first try connecting to https://doe.com:41145/, and fallback querying the SRV record _cap._tcp at 'doe.com' and use the specified server for the requests.
 
-HTTPS is required for all CAP endpoints, and as is standard the port 443 will be used.
-
-Endpoints are prefixed with Protocol version in case of non-backwards compatibility.
+## Endpoints
 
 <hr>
 
-## TIER 1 ENDPOINT
-
-The following GET endpoint is the only endpoint a server needs to support to be CAP 1 compliant. This is because in order for a user to have a CAP alias, wallets just need to know where to look up the address. CAP 1 can be as simple as a static string served from the endpoint.
-
-### GET /v1/address/{ledger_id}/{username}
+### GET /{domain}/{username}/{ledger_id}
 
 No Authorization (Public method)
 
-Returns the address of the related username and ledger, if it exists on the server.
+Returns the address of the related domain, username and ledger, if it exists on the server.
 
 #### Example Usage
 
-* Bob's decides to send Alice 5 BTC via her alias alice@ogdolo.com
-* Bob's wallet parses the alias into a URL destination and HTTP method
-* BTC_SEGWIT = ledger_id: 1
-* alice@ogdolo.com -> GET <https://opencap.ogdolo.com/address/1/alice>
-* If alice truly has a Bitcoin address hosted on the domain that her alias suggests, the address will be sent back to Bob's wallet with an HTTP 200 Status OK.
+* Bob decides to send Alice 5 BCH via her alias alice@domain.tld
+* Wallet finds a CAP server running at https://domain.tld:41145/
+* Ledger ID for Bitcoin Cash = 0
+* alice@domain.tld -> GET <https://domain.tld:41145/domain.tld/alice/0>
+* If Alice truly has a Bitcoin Cash address, the address will be sent back to Bob's wallet with an HTTP 200 Status OK.
 
 ```javascript
 Code: 200
 {
     "type": 0,
     "address": "YukHsVy/J9VCU5nr9vD7UOu4jxg=",
     "extensions": {
-        "dnssig": "..."
+        "name": "Satoshi Nakamoto",
+        "dns_sig": "..."
+        "pgp_sig": "..."
     }
 }
 ```
 
-* Bob's wallet can notify Bob that a valid address was found, and the wallet can now route a payment to Alice through her address.
-
-A wallet that uses multiple "ledgers" (BTC has legacy addresses and segwit addresses that constitute different ledgers within CAP) would first query the endpoint using the ledger_id of the preferred ledger, and if that fails it can try other ledgers. If the sender was able to send to a segwit address, they would first query the BTC_SEGWIT id (1) and if that fails, they would try the BTC_LEGACY id (0).
-
-<hr>
-
-## TIER 2 ENDPOINTS

```

Comment:
Where are all the her 2 endpoints going? We still need them.

Reply:
We'll need to discuss this. I'm not sure whether we want them to be part of the protocol.

Reply:
What?? That's very silly. We definitely need them, the bitcoin endpoints will be worthless without them.

Reply:
The sentiment on discord was that alias management shouldnt be part of the protocol. Websites will want to do their own thing. The most important thing is how people query addresses.

Reply:
Yeah thats why address queries are tier 1, I cannot get behind removing the tier 2 protocol though. It is necessary for BTC, and we can't move this protocol forward without BTC. Wallets should be able to use new addresses for each receive, its a privacy feature that we can't lose.

I haven't heard an arguement for why we wouldn't want support the feature, just for why they aren't necessary. I understand that they aren't absolutely necessary but there is no reason not to support the feature.

Reply:
I think you misunderstood. Removing them from the protocol does not mean they arent supported. As I said in discord, we can publish tier 2 and up as "possible implementation of alias management". The point is that there will be many teams that might want to do stuff a bit differently then we specified

Reply:
I deleted all the CUD operation stuff first so that we can agree on the endpoints level by level.

Reply:
Leave the tier 2 stuff in for now. We can discuss changing it in the next PR. We need to keep these PRs smaller anyhow.